### DISCUSSION TITLE
Front End Edit - option to hide Dock per Template

### DISCUSSION THREAD

**Discussion by ic360-com** [ExpressionEngine/ExpressionEngine#3623] [Feature Requests]
### Suggest an idea for ExpressionEngine

When "Front End Editing" is active... printing website pages (or pages like "Packing Sheet" Or "Invoice") outputs a rogue page with graphical elements from the Dock.
See attached PDF.
[Front End Edit.pdf](https://github.com/ExpressionEngine/ExpressionEngine/files/12105420/Front.End.Edit.pdf)

1) Is this fixable globally?
2) Would it be possible have a "Hide Dock" option per Template?

**Comment by intoeetive** [comment]
@ic360-com if you need fix now, please use files from this PR https://github.com/ExpressionEngine/ExpressionEngine/pull/3624
